test/SOFTSUSY/module.mk

```diff
@@ -115,11 +115,11 @@ $(LIBSOFTSUSY): $(LIBSOFTSUSY_OBJ)
 
 $(RUN_SOFTSUSY_EXE): $(RUN_SOFTSUSY_OBJ) $(LIBSOFTSUSY) $(MODSOFTSUSY_LIB) $(LIBFLEXI) $(filter-out -%,$(LOOPFUNCLIBS)) $(FUTILIBS)
 		@$(MSG)
-		$(Q)$(CXX) -o $@ $(call abspathx,$^) $(filter -%,$(LOOPFUNCLIBS)) $(FLIBS)
+		$(Q)$(CXX) -o $@ $(call abspathx,$^) $(filter -%,$(LOOPFUNCLIBS)) $(FLIBS) $(PYTHONLIBS) ${GSLLIBS} ${HIGGSTOOLSLIBS}
 
 $(RUN_SOFTPOINT_EXE): $(RUN_SOFTPOINT_OBJ) $(LIBSOFTSUSY) $(MODSOFTSUSY_LIB) $(LIBFLEXI) $(filter-out -%,$(LOOPFUNCLIBS)) $(FUTILIBS)
 		@$(MSG)
-		$(Q)$(CXX) -o $@ $(call abspathx,$^) $(filter -%,$(LOOPFUNCLIBS)) $(FLIBS)
+		$(Q)$(CXX) -o $@ $(call abspathx,$^) $(filter -%,$(LOOPFUNCLIBS)) $(FLIBS) $(PYTHONLIBS) ${GSLLIBS} ${HIGGSTOOLSLIBS}
```

Comment:
I think we should use parenthesis here, i.e.
~~~
$(GSLLIBS) $(HIGGSTOOLSLIBS)
~~~

Reply:
Fixed in debf2db59b3ce41e7fff09498d9ec68a7c5669f5.